shadow-cljs.edn

```diff
@@ -7,7 +7,7 @@
                 [binaryage/oops "0.7.0"]
                 [com.andrewmcveigh/cljs-time "0.5.2"]
                 [status-im/timbre "4.10.0-2-status"]
-                [com.taoensso/encore "2.105.0"]
+                [com.taoensso/encore "3.24.0"]
```

Comment:
could you pls elaborate on why this change is needed for this PR ?

Reply:
Oh, that's not necessary.
I saw it's not used else where. So I thought I'll just upgrade it to the latest version, which have bug fixes and improvements, and it won't break anything.

Do I need to change it back?

Reply:
yeah its better to rever it, thanks

Reply:
The version is reverted